spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/reactive/MultipartProperties.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2012-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.autoconfigure.web.reactive;
+
+import java.nio.charset.Charset;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.http.codec.multipart.DefaultPartHttpMessageReader;
+import org.springframework.util.unit.DataSize;
+
+/**
+ * {@link ConfigurationProperties Configuration properties} for configuring multipart
+ * support in Spring Webflux. Used to configure the {@link DefaultPartHttpMessageReader}.
+ *
+ * @author Chris Bono
+ * @since 2.6.0
+ */
+@ConfigurationProperties(prefix = "spring.webflux.multipart", ignoreUnknownFields = false)
+public class MultipartProperties {
+
+	/**
+	 * Maximum amount of memory allowed per part in bytes. Leave blank to use default of
+	 * 256K or set to -1 to store all contents in memory. Ignored when streaming is
+	 * enabled.
+	 */
+	private DataSize maxInMemorySize;
```

Comment:
What is the recommended approach here when we are configuring a target that already has defaults that we want to use? One one hand specifying  `private DataSize maxInMemorySize = DataSize.ofKilobytes(256);` the default here insulates us against any changes to the underlying target. However, it insulates us against any changes to the underlying target. Assuming we want to lock in-step w/ the underlying target this approach is fine.

Reply:
What we can do is set the default value in the properties class but write a test that checks that both defaults are aligned.

Reply:
I should have known this as I did that same thing when we aligned the CassandraProperties in another proposal a while back. :)